config/config.js

```diff
@@ -0,0 +1,30 @@
+import env from "../config/env.generated.json";
+
+let config = {
```

Comment:
Agree. i had a hard time naming that object. Do you thinking it's fine to name this file `config.js`?